tsl/src/planner.c

```diff
@@ -179,6 +180,34 @@ copy_mode_enabled(void)
 	return strcmp(enable_copy, "true") == 0;
 }
 
+/* Query tree walker to examine RTE contents for a distributed hypertable */
+static bool
+distributed_rtes_walker(Node *node, bool *isdistributed)
```

Comment:
Not sure what the return value represents. Could you document it?

Reply:
Added a comment